src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -230,21 +230,7 @@ void CommissionerDiscoveryController::InternalOk()
     {
         ChipLogDetail(AppServer, "UX InternalOk: app not installed.");
 
-        // notify client that app will be installed
```

Comment:
The lack of automated tests means that all of these changes are "blind" --> this is public API and may have been relied-on by participants (it's in `src/controller`, not `examples`), since `mUserPrompter` (provided by clients) is called. The lack of tests on this feature is very concerning.

Reply:
Yes. The proposal is to have tests that cover the features that are written :)

Reply:
Created an issue: https://github.com/project-chip/connectedhomeip/issues/34021